Olivia in Docker won't start

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to 'Portainer'
2. Click on 'Olivia'
3. Scroll down to 'start'
4. See an error in the log

**Expected behavior**
For it to start and be able to open the web app

**Desktop (please complete the following information):**
 - OS: [Windows 10 Pro]
 - Browser [Chrome]
 - Version [86.0.4240.183 (Official Build) (64-bit) (cohort: 86_Win_183]

**Additional context**
Here is the Log 

**       _ _       _,
  ___ | (_)_   _(_) __ _,
 / _ \| | \ \ / / |/ _` |,
| (_) | | |\ V /| | (_| |,
 \___/|_|_| \_/ |_|\__,_|,
,
hey,
Your authentication token is: b60abbbb5d88770c3f255c2856bda3721b63ae434caf12ff3c6bb2035972,
Save it, you won't be able to get it again unless you generate a new one.,
,
Loading the neural network from res/locales/en/training.json,
Loading the neural network from res/locales/de/training.json,
Loading the neural network from res/locales/fr/training.json,
Loading the neural network from res/locales/es/training.json,
Loading the neural network from res/locales/ca/training.json,
Loading the neural network from res/locales/it/training.json,
Loading the neural network from res/locales/tr/training.json,
Loading the neural network from res/locales/nl/training.json,
Loading the neural network from res/locales/el/training.json,
,
Server listening on the port $PORT...,
panic: listen tcp: address tcp/$PORT: unknown port,
,
goroutine 1 [running]:,
github.com/olivia-ai/olivia/server.Serve(0xc00040b860, 0x7ffeccfe3f4d, 0x5),
	/app/server/serve.go:54 +0x6b9,
main.main(),
	/app/main.go:57 +0x425,**

--- a/README.md
+++ b/README.md
@@ -92,7 +92,7 @@ $ docker pull docker.pkg.github.com/olivia-ai/olivia/olivia:latest
 
 Then start it
 ```bash
-$ docker run -d -p 8080:8080 docker.pkg.github.com/olivia-ai/olivia/olivia:latest
+$ docker run -d -e PORT=8080 -p 8080:8080 docker.pkg.github.com/olivia-ai/olivia/olivia:latest
 ```
 
 You can just use the websocket of Olivia now.

